test/client_serverWS_test.jl

```diff
@@ -50,13 +51,15 @@ let
     WebSockets.open(initiatingws, "ws://$SURL:$PORT")
     close(server)
 end
+sleep(1)
```

Comment:
Move inside let block? We should not be surprised that a 'sleep' is required. 'yield' would probably be sufficient and not take extra time.